lib/SearchShortcuts.jsm

```diff
@@ -8,15 +8,20 @@
 const SEARCH_SHORTCUTS = [
```

Comment:
I had to take a slightly different approach here since ursula's patch would need to be updated if I included a new property with which to filter the list